src/engraving/libmscore/edit.cpp

```diff
@@ -1869,6 +1869,10 @@ void Score::cmdAddOttava(OttavaType type)
             }
             Ottava* ottava = new Ottava(this);
             ottava->setOttavaType(type);
+            if (type == OttavaType::OTTAVA_8VB/* || type == OttavaType::OTTAVA_15MB || type == OttavaType::OTTAVA_22MB*/) {
```

Comment:
No need to check for `OttavaType::OTTAVA_15MB` and `OttavaType::OTTAVA_22MB`, as there's no shortcut for those

Reply:
But logically the user should be able to define such shortcuts ( and you probably can by editing files directly), so I'd think we should support that anyway?
More importantly, shouldn't this logic belong inside Ottava itself? (i.e. setOttavaType( ) should set the "placement" property as appropriate)

Reply:
I don't think you can just invent new shortcuts without code changes. Directly editing the shortcuts.xml won't work, as far as I can tell.
Yes. that decision, above/below, might better be taken elsewhere, something for another PT maybe, here I just tried to keep the changes as minimal as possible

Reply:
Hmm, maybe that `cmdAddOttava()` can get used from whithin plugins, and then with arguments other than just `OttavaType::OTTAVA_8VA` and `OttavaType::OTTAVA_8VB`.

Reply:
Edit: no, It can't, only the  "add-8va" and "add-8vb" from cmd.cpp